src/span_context.js

```diff
@@ -68,7 +68,12 @@ export default class SpanContext {
 
   get traceId(): any {
     if (this._traceId == null && this._traceIdStr != null) {
-      this._traceId = Utils.encodeInt64(this._traceIdStr);
+      const safeTraceIdStr = this._traceIdStr.length % 2 == 0 ? this._traceIdStr : '0' + this._traceIdStr;
+      const tmpBuffer = new Buffer(safeTraceIdStr, 'hex');
+      const size = tmpBuffer.length > 8 ? 16 : 8;
+      this._traceId = new Buffer(size);
```

Comment:
It seems as `new Buffer` [is deprecated](https://nodejs.org/api/buffer.html#buffer_new_buffer_array). We should probably use the new API. So this line and the subsequent line can jointed into:
```
this._traceId = Buffer.alloc(size)
```
Then we can rely on babel to provide backwards compatibility.

Reply:
@everett980 I don't think babel will provide backwards compatibility for the `Buffer.from` because it's not a syntax issue.